entrypoint.sh

```diff
@@ -115,20 +115,27 @@ main() {
   aws --version
 
   # Iterate over $INPUT_SOURCE multiline string and run aws s3 $COMMAND
-  while IFS= read -r source; do
+while IFS= read -r source; do
     if [ -n "$source" ]; then
-        if [ "$COMMAND" == "cp" ] || [ "$COMMAND" == "mv" ] || [ "$COMMAND" == "sync" ]
-        then
-            echo "aws s3 $COMMAND \"$source\" $INPUT_DESTINATION $INPUT_FLAGS"
-            aws s3 "$COMMAND" "$source" "$INPUT_DESTINATION" "$INPUT_FLAGS"
+        if [ "$COMMAND" == "cp" ] || [ "$COMMAND" == "mv" ] || [ "$COMMAND" == "sync" ]; then
+            if [ -n "$INPUT_FLAGS" ]; then
+                cmd="aws s3 $COMMAND \"$source\" $INPUT_DESTINATION $INPUT_FLAGS"
```

Comment:
Just a suggestion - let's [replace spaces with underscores.](https://stackoverflow.com/a/19661428) Should eliminate majority of cases.
```suggestion
            if [ -n "$INPUT_FLAGS" ]; then
                source="${source// /_}"
                cmd="aws s3 $COMMAND \"$source\" $INPUT_DESTINATION $INPUT_FLAGS"
```